apps/meteor/ee/app/livechat-enterprise/server/api/priorities.ts

```diff
@@ -27,7 +27,7 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'livechat/priorities/:priorityId',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: { GET: { permissions: ['manage-livechat-priorities', 'view-l-room'], operation: 'hasAny' } } },
```

Comment:
Since we have a permission check here, so can we remove it from `findPriorityById` to avoid re-checking?